### ISSUE TITLE
Template is outdated

### ISSUE THREAD

**Issue by vaxul** [platformsh-templates/typo3#32]
Just an opinion. But this template should be update at least to a TYPO3 v11 instance. Maybe it would be more appropriate to tag multiple TYPO3 versions (10, 11, 12, ...).

**Comment by rfay** [comment]
Yes, this seems to be TYPO3 v10, and current LTS is v12. This one should probably be retired.
